superset-frontend/spec/javascripts/components/URLShortLinkButton_spec.jsx

```diff
@@ -20,7 +20,7 @@ import React from 'react';
 import configureStore from 'redux-mock-store';
 import { shallow } from 'enzyme';
 
-import { OverlayTrigger } from 'react-bootstrap';
+import Popover from 'src/common/components/Popover';
```

Comment:
please use `src/components/Popover`, which for now is just a redirection to 'src/common/components/Popover', but will allow to eventually alter/style all popover form that central point of convergence.

Reply:
Other components from Antd are placed in `src/common/components` and they are wrapped in `styled` from emotion. I think moving Popover to `src/components/` would lead to inconsistency.

Reply:
Oh I see, there's 2 pockets emerging here. @rusackas can clarify how this should evolve. Maybe thin wrappers (style only) are in `common/components` (?). Ultimately the "design system" and vetted components should all live in the same place. Maybe it's a matter of moving `src/common/components` to `src/components` at some point, or vice-versa. 

This discussion shouldn't hold this PR back, it's easy to move/reorganize things later on.

Reply:
👍

Reply:
Agree that this discussion shouldn't hold the PR back, but here are my thoughts, in a nutshell:

1) For components where we're just adding Emotion styling to AntD components, we should export them from their own modules, within the common folder (e.g. /src/common/components/Whatever), where the component is in its own Folder i.e.:
```
└─┬ Widget
  └── index.tsx
  └── Widget.stories.tsx
```
Then, _IF_ we need to add new functionality to one of these components, it's all self-contained, and can be moved into the main `/src/components/` folder, where we would then add a `Widget.test.tsx` file to the directory to cover (at least) the new functionality.

Let me know if this sounds good. If so, I'll happily shuffle around the existing "painted" AntD components to match this pattern, and add some info to the `CONTRIBUTING` file.